docs/faq/code-analysis/which-metrics-does-codacy-calculate.md

```diff
@@ -55,7 +55,7 @@ Codacy calculates the number of issues in the following static code analysis cat
 -   **Performance:** Code that can have performance problems
 -   **Compatibility:** Mainly for frontend code, compatibility problems across different browser versions
 -   **Unused Code:** Unused variables and methods, code that can't be reached
--   **Security:** All security problems
+-   **Security:** Potential security vulnerabilities, including hardcoded passwords and keys (secret scanning), vulnerable dependencies (software composition analysis or SCA), and insecure code patterns (static application security testing or SAST). For more information, see the complete [list of security issue categories](../../repositories/security-monitor.md#supported-security-categories)
```

Comment:
Hyphenating to fix Vale issue, as per [MS style guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/h/hard-code-hard-coded-hard-coding).

```suggestion
-   **Security:** Potential security vulnerabilities, including hard-coded passwords and keys (secret scanning), vulnerable dependencies (software composition analysis or SCA), and insecure code patterns (static application security testing or SAST). For more information, see the complete [list of security issue categories](../../repositories/security-monitor.md#supported-security-categories)
```